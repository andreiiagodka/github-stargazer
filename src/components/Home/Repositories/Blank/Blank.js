import React from 'react'
import { Container, Row, Col } from 'react-bootstrap'

const blank = props => (
  <Row>
    <Col md={12} className='text-center'>
      <p className='lead'>No repositories added</p>
    </Col>
  </Row>
)

export default blank