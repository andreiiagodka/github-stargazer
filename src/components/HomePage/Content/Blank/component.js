import React from 'react'

import { Row, Col } from 'react-bootstrap'

const BlankComponent = () => (
  <Row>
    <Col md={12} className='text-center'>
      <p className='lead'>No repositories added</p>
    </Col>
  </Row>
)

export default BlankComponent