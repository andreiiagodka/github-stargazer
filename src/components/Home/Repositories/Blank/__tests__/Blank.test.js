import React from 'react'
import renderer from 'react-test-renderer'

import Blank from '../Blank'

describe('<Blank /> component', () => {
  it('renders component correctly', () => {
    const tree = renderer.create(<Blank />).toJSON()
  
    expect(tree).toMatchSnapshot()
  })
})
