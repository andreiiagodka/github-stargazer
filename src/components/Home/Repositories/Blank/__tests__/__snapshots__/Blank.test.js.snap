// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Blank /> component renders component correctly 1`] = `
<div
  className="row"
>
  <div
    className="text-center col-md-12"
  >
    <p
      className="lead"
    >
      No repositories added
    </p>
  </div>
</div>
`;
